utils/src/main/java/org/jboss/windup/util/threading/WindupExecutors.java

```diff
@@ -0,0 +1,20 @@
+package org.jboss.windup.util.threading;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+/**
+ * An alterantive to {@link Executors} that pushes windup specific threadFactory inside by default. This is useful in order to have
+ * be able to connect sibling threads into the main thread of windup run.
+ */
+public class WindupExecutors
+{
+    public static ExecutorService newFixedThreadPool(int numberOfThreads) {
+        return Executors.newFixedThreadPool(numberOfThreads,new WindupChildThreadFactory());
```

Comment:
I am open to any suggestions. However isn't it better to discard all the threads once we are done with the work? And then recreate it?